saleor/discount/models.py

```diff
@@ -130,13 +130,18 @@ class Meta:
     @property
     def code(self):
         # this function should be removed after field `code` will be deprecated
+        # use last_code instead
         code_instance = self.codes.last()
         return code_instance.code if code_instance else None
 
     @property
     def promo_codes(self):
         return list(self.codes.values_list("code", flat=True))
 
+    @property
+    def last_code(self):
+        return self.promo_codes[-1] if self.promo_codes else None
```

Comment:
We want to get rid of `code`, so we shouldn't add a new property that is basically the same as what the `code` is. We should get rid of all usage of `code` and not add any new.